### PR TITLE
Lei-Liu's first pull request

### PR DESCRIPTION
**Pull request by LeiLiu1991**

### CODE REVIEW

File: Lei-Liu [hunk 1]
```diff
@@ -0,0 +1 @@
+I want to merge this file.
```
Comment (Alvin-Lau) [inline Lei-Liu]: Would you please explain why this is needed?

Comment (LeiLiu1991) [inline Lei-Liu]: This is a new feature

Comment (Alvin-Lau) [inline Lei-Liu]: Please fix this issue

File: Lei-Liu [hunk 2]
```diff
@@ -0,0 +1,2 @@
+I want to merge this file. 
+Fix a issue
```
Comment (Alvin-Lau) [inline Lei-Liu]: Yes, this works

**Pull request merged**